src/exercises/inputNormalization.js

```diff
@@ -0,0 +1,15 @@
+import removeAccents from "remove-accents";
+
+function removeQuotes(x) {
+  return x.replace(/[^a-zA-Z ]/g, "");
+}
+
+function normalizeCase(x) {
+  return x.trim().toUpperCase();
+}
+
+function normalizeWord(x) {
```

Comment:
@tfnribeiro - should this be `normalizeAnswer`? We don't use it in any other case but for checking exercise answers...

Reply:
Considering the actions we are taking, I think it's unlikely we would use it anywhere else, especially since we are normalizing case too. This is where maybe we could add a bit more logic in the future where e.g. for German we maintain capitalization.

Reply:
I completely agree with you. And I'll take your answer as meaning - yes, let's rename to `normalizeAnswer` then :) I'll do that and then merge the PR.